content/en/blog/_posts/2020-04-21-contributor-communication-upstream-marketing.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: blog
+title: How Kubernetes contributors are building a better communication process
+date: 2020-04-21
+slug: contributor-communication
+---
+
+**Authors:** Paris Pittman
+
+> “Perhaps we just need to use a different word. We may need to use community development or project advocacy as a word in the open source realm as opposed to marketing, and perhaps then people will realize that they need to do it.” 
+> ~ [*Nithya Ruff*](https://todogroup.org/www.linkedin.com/in/nithyaruff/) (from [*TODO Group*](https://todogroup.org/guides/marketing-open-source-projects/))
+
+A common way to participate in the Kubernetes contributor community is
+to be everywhere. 
+
+We have an active Slack, many mailing lists, Twitter account(s), and
+dozens of community-driven podcasts and newsletters that highlight all
+end-user, contributor, and ecosystem topics. And guess what else - an
+entire repository of amazing documentation that I bet there is one gem
+you didn’t know we had! All of this information is incredibly valuable,
+but it can be too much. 
+
+Keeping up with thousands of contributors can be a challenge for anyone,
+but this task of consuming information straight from the firehose is
+particularly challenging for new community members. It’s no secret that
+the project is vast for contributors and users alike. 
+
+To paint a picture with numbers: 
+
+- 43,000 contributors
+- 6,579 members in #kubernetes-dev slack channel
+- 52 mailing lists (kubernetes-dev@ has thousands of members; sig-networking@ has 1000 alone)
+- 40 [community groups](https://github.com/kubernetes/community/blob/master/governance.md#community-groups)
+- 30 [upstream meetings](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America%2FLos_Angeles) *this* week alone
+
+The scale is growing, and with that comes the need for attention.
+
+## How we got here
+
+Kubernetes (k8s for short) communication grew out of a need for people
+to connect in our growing community. With the best of intentions, the
+community spun up channels for people to connect. This energy was part
+of what helped Kubernetes grow so fast, and it also had us in a space of
+sprawling out far and wide. As adoption grew, [contributors knew there was a need for standardization](https://github.com/kubernetes/community/issues/2466).
+
+This new attention to how the community communicates led to a discovery
+of a complex web of options. There were so many options, it was a
+challenge for anyone to be sure they were in the right place to receive
+the right information. We started taking immediate action by pruning the
+inactive mailing lists and other platforms, combining where we could,
```

Comment:
nit: combining what?

Reply:
mailing lists and other forums (reducing redundancy, stale, etc)

Reply:
Thinking of something like:
  ... combining communication streams [or channels] and thinking  
about how to [best reach out to [serve] our community?].

Reply:
That sentence was ambiguous - added "them" to flow.